pkgs/servers/matrix-synapse/matrix-appservice-slack/default.nix

```diff
@@ -8,22 +8,39 @@ let
   };
 
   nodePackages = import ./node-composition.nix {
-    inherit pkgs nodejs;
+    inherit pkgs;
     inherit (stdenv.hostPlatform) system;
+    nodejs = nodejs-slim;
   };
+  pname = "matrix-appservice-slack";
 in
 nodePackages.package.override {
-  pname = "matrix-appservice-slack";
 
   inherit src;
```

Comment:
Shouldn't this also include `pname` now that you moved it to the `let` above?

Reply:
The name part is already handled by `node2nix` so assigning the `pname` only really makes sense if you want to rename the package. In this case, it is more or less a no-op, so I moved it to the `let` block to have a nicer test name.